src/v2/view-builder/views/TerminalView.js

```diff
@@ -3,13 +3,14 @@ import BaseForm from '../internals/BaseForm';
 import BaseFooter from '../internals/BaseFooter';
 
 const Body = BaseForm.extend({
-  attributes: { 'data-se': 'terminal-state' },
-
   title () {
     const msg = this.options.appState.get('terminal').message || {};
     return msg.message || 'You can close this window';
   },
   noButtonBar: true,
+  postRender () {
+    this.$el.addClass('terminal-state');
```

Comment:
shall we invoke `super.postRender`? although nothing happens at super now but to be consistent about how to override method.